### PR TITLE
2021 10 04 issue 3715

### PR DESCRIPTION
**Pull request by Christewart**
Test case to try and reproduce #3715

Unfortunately I am unable to reproduce, would like eyes from @benthecarman to make sure i'm not missing something.

### CODE REVIEW

**Comment by benthecarman** [comment]
In `createFundedDLCWalletWithBitcoind` you should call `startBitcoindBlockPolling` so you can simulate the exact thing the appServer is doing

**Comment by Christewart** [comment]
> In `createFundedDLCWalletWithBitcoind` you should call `startBitcoindBlockPolling` so you can simulate the exact thing the appServer is doing

I don't think this is relevant. We are failing to broadcast from **our** wallet to bitcoind. Not syncing it from external chain data.

File: testkit-core/.jvm/src/main/resources/logback-test.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
         </encoder>
     </appender>
 
-    <root level="OFF">
+    <root level="ERROR">
```
Comment (benthecarman) [inline testkit-core/.jvm/src/main/resources/logback-test.xml]: shuold revert

File: testkit/src/main/scala/org/bitcoins/testkit/wallet/BitcoinSDualWalletTest.scala [hunk 2]
```diff
@@ -61,41 +66,127 @@ trait BitcoinSDualWalletTest extends BitcoinSWalletTest {
...
+        extraConfig = Some(segwitWalletConf))
+      walletB <- FundWalletUtil.createFundedDLCWallet(
+        nodeApi = nodeApi,
+        chainQueryApi = chainQueryApi,
+        bip39PasswordOpt = getBIP39PasswordOpt(),
+        extraConfig = Some(segwitWalletConf))(config2, system)
+      _ = println(s"---------------------------------")
+      amt = expectedDefaultAmt / Satoshis(2)
+      contractInfo = ContractInfo(amt.satoshis, contractOraclePair)
+      _ = println(s"@@@@@@@@@@@@@@@@@@@@@@@@@@")
```
Comment (benthecarman) [inline testkit/src/main/scala/org/bitcoins/testkit/wallet/BitcoinSDualWalletTest.scala]: printlns

File: testkit/src/main/scala/org/bitcoins/testkit/wallet/FundWalletUtil.scala [hunk 3]
```diff
@@ -78,6 +76,17 @@ trait FundWalletUtil extends Logging {
...
 
+  /** Funds a bitcoin-s wallet with 3 utxos, 1,2,3 bitcoin in the utxos
+    * that are funded by REAL utxos from the given bitcoind.
+    * Other test methods such as [[fundWallet()]] create fictitious utxos
+    * that doesn't correspond to a live network.
+    */
+  def fundWalletWithBitcoind(
+      wallet: Wallet,
+      bitcoind: BitcoindRpcClient): Future[FundedTestWallet] = {
+    ???
```
Comment (benthecarman) [inline testkit/src/main/scala/org/bitcoins/testkit/wallet/FundWalletUtil.scala]: should implement or delete

**Review by benthecarman — APPROVED** [review]

**Pull request merged**